packages/nimble-components/src/table-column/base/tests/table-column-stories-utils.ts

```diff
@@ -0,0 +1,36 @@
+import type { Table } from '../../../table';
+import type { TableRecord } from '../../../table/types';
+
+export interface CommonTableArgs {
+    tableRef: Table;
+    updateData: (args: CommonTableArgs) => void;
+}
+
+export const commonTableArgTypes = {
+    tableRef: {
+        table: {
+            disable: true
+        }
+    },
+    updateData: {
+        table: {
+            disable: true
+        }
+    }
+};
+
+export const commonTableArgs = (
```

Comment:
Can you follow a pattern similar to the `sharedMatrixParameters` pattern.
In-specific:
- have the resulting object be `as const`
- Don't use the shared value directly but "merge" it. ie the Args type should extend the CommonTableArgs type, spread the common table args into args, spread the common tableargstypes into argtypes, etc. This makes it easier for future story authors to see where to insert new args, etc.
- Be consistent with the naming convention, i.e. the shared prefix.